pkg/virt-controller/watch/vm_test.go

```diff
@@ -154,6 +154,12 @@ var _ = Describe("VirtualMachine", func() {
 			vmiInterface.EXPECT().Patch(vmi.Name, types.JSONPatchType, []byte(patch), &metav1.PatchOptions{}).Return(vmi, nil)
 		}
 
+		shouldExpectVMFinalizerRemoval := func(vm *virtv1.VirtualMachine) {
+			patch := `[{ "op": "test", "path": "/metadata/finalizers", "value": ["kubevirt.io/virtualMachineControllerFinalize"] }, { "op": "replace", "path": "/metadata/finalizers", "value": [] }]`
```

Comment:
Can you please use the const `virtualMachineControllerFinalize` here instead of the string literal?

Reply:
Done